types/sortablejs/sortablejs-tests.ts

```diff
@@ -424,6 +424,47 @@ const { AutoScroll, MultiDrag, OnSpill, Swap } = Sortable;
 Sortable.mount(new AutoScroll(), new MultiDrag(), new OnSpill(), new Swap());
 Sortable.mount(new MultiDrag());
 
+// AutoScroll
+new Sortable(el, {
+	scroll: true,
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/sortablejs/sortablejs-tests.ts;
  Line: 429;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.